HeapSort_Py/HeapSort.py

```diff
@@ -0,0 +1,13 @@
+def HeapSort(arr):
+    for i in range(len(arr), 1, -1):
+        parent = arr[i//2 - 1]
+        child = arr[i - 1]
+        
+        print('Parent of ' + str(child) + ' : ' + str(parent))
+
+        if(parent < child):
+            arr[i//2 - 1], arr[i - 1] = child, parent
+    return arr
+
+arr = [20,10,6,2,5,3,15]
+print(HeapSort(arr))
```

Comment:
Are you sure this algorithm is complete?
Because the output array is not sorted.

Reply:
Yes, the algorithm is complete, it gives a tree as an output in the form of Array.
Considering the arr given in program, 20 is the root, with two children, 10 and 6, and 10 have two children, 2 and 5, and so on. 
Though it is limited to parents with only two children, it does sort the tree, where each parent's left is smaller than itself and the right will be bigger.

Reply:
The output for your example is [20, 10, 15, 2, 5, 3, 6].
If we want to form the tree then 20 will be the root node and 10 will be its left child and 15 will be right. But 15 is smaller than 20 which should be greater if it's sorted.

Reply:
Yes, you are right, it is not working as expected.
Thanks for the review, will remove it and add it later when complete.